### PR TITLE
GH-483: Add KafkaAdmin operationTimeout

### PR DESCRIPTION
**Pull request by garyrussell**
Fixes https://github.com/spring-projects/spring-kafka/issues/483

__cherry-pick to 2.0.x, 1.3.x__

### CODE REVIEW

**Comment by garyrussell** [comment]
Only first commit can be cherry-picked.

File: spring-kafka/src/main/java/org/springframework/kafka/core/KafkaAdmin.java [hunk 1]
```diff
@@ -91,6 +97,14 @@ public void setCloseTimeout(int closeTimeout) {
 	}
 
 	/**
+	 * Set the operation timeout in seconds. Defaults to 30 seconds.
+	 * @param closeTimeout the timeout.
```
Comment (artembilan) [inline spring-kafka/src/main/java/org/springframework/kafka/core/KafkaAdmin.java]: I haven't polled it locally yet, but I know that this one is going to be red for my in the IDEA 

The real method param is `operationTimeout`, not `closeTimeout `

😄

**Comment by artembilan** [comment]
Oh, yeah! Travis confirms my concern:
```
[ant:checkstyle] [ERROR] /home/travis/build/spring-projects/spring-kafka/spring-kafka/src/main/java/org/springframework/kafka/core/KafkaAdmin.java:101:12: Unused @param tag for 'closeTimeout'. [JavadocMethod]

[ant:checkstyle] [ERROR] /home/travis/build/spring-projects/spring-kafka/spring-kafka/src/main/java/org/springframework/kafka/core/KafkaAdmin.java:103:45: Expected @param tag for 'operationTimeout'. 
```
Time to switch to IDEA or open bug request to the Eclipse 😉

**Comment by artembilan** [comment]
Merged as https://github.com/spring-projects/spring-kafka/commit/bf734278a613b9e2ece8048df7685a24095bfdda after rebasing and squashing.

Cherry-picked (first commit) to `2.0.x` as https://github.com/spring-projects/spring-kafka/commit/9eea2f8e2b4fedb1990316160d5c01afd8805d29 and to `1.3.x` as https://github.com/spring-projects/spring-kafka/commit/e7765192ddb04cb2862b9e0f8d7308cbfcb2606c after resolving conflicts.

**Pull request closed without merging**